vscode/microsoft-kiota/src/utilities/workspace.ts

```diff
@@ -0,0 +1,11 @@
+
+import * as vscode from 'vscode';
+
+export function isPathInWorkspace(filePath: string): boolean {
```

Comment:
Is this being used anywhere?

Reply:
No. I changed the concept before the last commit

Reply:
Do we need to remove the file then in that case? No need maintaining an unused code.